migrations/versions/0259_service_volumes.py

```diff
@@ -0,0 +1,30 @@
+"""
+
+Revision ID: 0259_service_volumes
+Revises: 0258_service_postage_nullable
+Create Date: 2019-02-13 13:45:00.782500
+
+"""
+from alembic import op
+from itertools import product
+import sqlalchemy as sa
+
+
+revision = '0259_service_volumes'
+down_revision = '0258_service_postage_nullable'
+
+
+TABLES_AND_CHANNELS = product(
+    ('services', 'services_history'),
+    ('volume_{}'.format(channel) for channel in ('email', 'letter', 'sms')),
+)
+
+
+def upgrade():
+    for table, channel in TABLES_AND_CHANNELS:
+        op.add_column(table, sa.Column(channel, sa.String(length=255), nullable=True))
```

Comment:
Why a string?

Reply:
There is already a 0259 migration :(

Reply:
@servingUpAces changed to integer in 68ed2c5

Reply:
@quis sorry there's now a 260 migration too

Reply:
Bumped version in 7ca62ce